### ISSUE TITLE
Can't get the desired experimental results using only aurora2 training set?

### ISSUE THREAD

**Issue by jujiubobo** [ClausMeyerLarsen/VAD-with-adversarial-multi-task-learning#1]
Only train on aurora2 training set and then test on aurora2 test set? Or train the model on two training sets? Why can't I get your lab results?
Best wish!
